src/cmd/manage_nsfs.js

```diff
@@ -275,7 +275,7 @@ async function delete_bucket(data, force) {
  */
 async function bucket_management(action, user_input) {
     const data = action === ACTIONS.LIST ? undefined : await fetch_bucket_data(action, user_input);
-    await manage_nsfs_validations.validate_bucket_args(config_fs, data, action);
+    await manage_nsfs_validations.validate_bucket_args(config_fs, data, action, user_input);
```

Comment:
Can we avoid adding a new argument `user_input` in the function signature?

Reply:
I didn't find a way.
The new input is required, the merged is not good enough.
When bucket name is changed, it is passed through a specific parameter ("new_name").

Reply:
But I don't see where you use the bucket name exactly.
Another option is maybe to check it later in the flow.

Reply:
Maybe you can add the `notifications` argument only? (I want to avoid confusion between `data` and `user_input`).